scripts/tests/test_migrate_metadata.py

```diff
@@ -0,0 +1,145 @@
+        'item5': 'fafa',
+        'item6': 'asdfsadf',
+        'item7': 'sfsaf',
+        'item8': 'sfdsdf',
+        'item9': 'sfd',
+    },
+    'Replication Recipe (Brandt et al., 2013): Post-Completion': {
+        'item29': 'adad',
+        'item30': 'asd',
+        'item31': 'asd',
+        'item32': 'not significantly different from the original effect size',
+        'item33': 'informative failure to replicate',
+        'item34': 'asdasd',
+        'item35': 'ds',
+        'item36': 'ads',
+        'item37': 'das',
+    },
+}
+
+
+class TestMigrateSchemas(OsfTestCase):
+    def setUp(self):
+        super(TestMigrateSchemas, self).setUp()
+
+        MetaSchema.remove()
+        ensure_schemas()
+
+        self.open_ended_schema = MetaSchema.find_one(
+            Q('name', 'eq', SCHEMA_NAMES[0]) &
+            Q('schema_version', 'eq', 1)
+        )
+        self.open_ended = factories.RegistrationFactory(
+            registered_meta={
+                to_mongo(SCHEMA_NAMES[0]): json.dumps(OLD_META[SCHEMA_NAMES[0]])
+            },
+            registered_schema=self.open_ended_schema
+        )
+        del self.open_ended.registered_meta['Template1']
+        self.open_ended.save()
+        self.standard_schema = MetaSchema.find_one(
+            Q('name', 'eq', SCHEMA_NAMES[1]) &
+            Q('schema_version', 'eq', 1)
+        )
+        self.standard = factories.RegistrationFactory(
+            registered_meta={
+                to_mongo(SCHEMA_NAMES[1]): json.dumps(OLD_META[SCHEMA_NAMES[1]])
+            },
+            registered_schema=self.standard_schema
+        )
+        del self.standard.registered_meta['Template1']
+        self.standard.save()
+        self.brandt_pre_schema = MetaSchema.find_one(
+            Q('name', 'eq', SCHEMA_NAMES[2]) &
+            Q('schema_version', 'eq', 1)
+        )
+        self.brandt_pre = factories.RegistrationFactory(
+            registered_meta={
+                to_mongo(SCHEMA_NAMES[2]): json.dumps(OLD_META[SCHEMA_NAMES[2]])
+            },
+            registered_schema=self.brandt_pre_schema
+        )
+        del self.brandt_pre.registered_meta['Template1']
+        self.brandt_pre.save()
+        self.brant_post_schema = MetaSchema.find_one(
+            Q('name', 'eq', SCHEMA_NAMES[3]) &
+            Q('schema_version', 'eq', 1)
+        )
+        self.brandt_post = factories.RegistrationFactory(
+            registered_meta={
+                to_mongo(SCHEMA_NAMES[3]): json.dumps(OLD_META[SCHEMA_NAMES[3]])
+            },
+            registered_schema=self.brant_post_schema
+        )
+        del self.brandt_post.registered_meta['Template1']
+        self.brandt_post.save()
+
+    def test_get_old_registered_nodes(self):
+        # create a non-registered node
+        factories.NodeFactory()
+
+        old_nodes = get_old_registered_nodes(dry_run=False)
+        assert_equal(len(old_nodes), 4)
+
+    def test_migrate_registration_schemas(self):
```

Comment:
These tests fail when run locally- as part of the responses to review, please rerun this file and make sure tests are passing. @samchrisinger indicates that `RegistrationFactory` was recently updated; I had some sporadic trouble running these tests on my computer.